models/demos/squeezebert/tests/test_perf_device_squeezebert.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from models.utility_functions import is_grayskull
+from models.perf.device_perf_utils import run_device_perf, check_device_perf, prep_device_perf_report
+
+
+@pytest.mark.models_device_performance_bare_metal
+@pytest.mark.parametrize(
+    "batch_size, test",
+    [
+        [8, "sequence_size=384-batch_size=8-model_name=squeezebert/squeezebert-uncased"],
+    ],
+)
+def test_perf_device_bare_metal(batch_size, test):
+    subdir = "ttnn_squeezebert"
+    num_iterations = 1
+    margin = 0.03
+    expected_perf = 114.8 if is_grayskull() else 284.5
```

Comment:
what;'s the current device perf we are getting? how close is it to expected_perf?

Reply:
The current expected_perf in the test file have been updated based on the device perf observed during CI runs.